client/app/vps/modal/backup-storage/delete-backup-storage.controller.js

```diff
@@ -0,0 +1,32 @@
+class DeleteBackupStorageCtrl {
+    constructor ($translate, $uibModalInstance, CloudMessage, access, serviceName, VpsService) {
+        this.$translate = $translate;
+        this.$uibModalInstance = $uibModalInstance;
+        this.CloudMessage = CloudMessage;
+        this.access = access;
+        this.serviceName = serviceName;
+        this.VpsService = VpsService;
+
+        this.loader = {
+            save: false
+        };
+    }
+
+    cancel () {
+        this.$uibModalInstance.dismiss();
+    }
+
+    confirm () {
+        this.loader.save = true;
+        this.VpsService.deleteBackupStorageAccess(this.serviceName, this.access)
+            .then(() => this.CloudMessage.success(this.$translate.instant("vps_backup_storage_access_delete_success")))
```

Comment:
I see that you never use successHandler nor errorHandler from service-helper.service.js .
Maybe in a second pass?

Reply:
Same thing for helpers in controller-request-helper.service.js